plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -84,6 +88,36 @@ class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer,
     onIndicatorBearingChangedListener.remove(listener)
   }
 
+  /**
+   * Check is a given point is on the rendered location puck.
+   *
+   * @param point the point to validate
+   * @param listener Listener that gets invoked when the validation finished.
+   */
+  override fun isPointOnLocationPuck(point: Point, listener: IsPointOnLocationPuckListener) {
+    delegateProvider.mapFeatureQueryDelegate.queryRenderedFeatures(
+      delegateProvider.mapProjectionDelegate.pixelForCoordinate(point),
+      RenderedQueryOptions(
+        listOf(
+          LOCATION_INDICATOR_LAYER,
+          MODEL_LAYER
+        ),
+        Value("all")
```

Comment:
just for curiosity - is passing `null` here equivalent to `Value("all")`?

Reply:
Good catch, double checked it seems passing null should be more appropriate here, since the filter should be an Expression and `Value("all")` is not actually an expression.